packages/aws-cdk-lib/README.md

```diff
@@ -0,0 +1,51 @@
+# AWS Cloud Development Kit Library
+
+[![experimental](http://badges.github.io/stability-badges/dist/experimental.svg)](http://github.com/badges/stability-badges)
+
+The AWS CDK construct library provides APIs to define your CDK application and add
+CDK constructs to the application.
+
+## Usage
+
+### Upgrade from CDK 1.x
+
+When upgrading from CDK 1.x, remove all dependencies to individual CDK packages
+from your dependencies file and follow the rest of the sections.
+
+### Installation
+
+Add a single entry in your dependencies list to this package.
+
+You also need to add a reference to the `constructs` library, according to the
+kind of project you are developing:
```

Comment:
Isn't this true for `aws-cdk-lib` itself? Libraries should declare peer dependency on both `aws-cdk-lib` and `constructs`

Reply:
Isn't this what lines 17-22 are saying, read all together. Is it not clear?

Reply:
This is the part that is confusing:
>Add a single entry in your  **dependencies** list to this package
It reads like one should add **this** package to the **dependencies**  clause of the `package.json` file.

Reply:
I've reworded. It should be clearer now.